src/nova_basic_handler.erl

```diff
@@ -173,8 +173,8 @@ handle_redirect({redirect, Route}, _ModFun, Req) ->
 -spec handle_sendfile({sendfile, StatusCode :: integer(), Headers :: map(), {Offset :: integer(),
                                                                              Length :: integer(),
                                                                              Path :: list()}, Mime :: binary()},
-                  ModFun :: mod_fun(), Req) -> {ok, Req} when Req :: cowboy_req:req().
```

Comment:
This will not work since a http-handler expects arity 3 of the handler-fnctions

Reply:
ah, ok and ws handler in cowboy is using arity 2?